src/main/resources/Model.vm

```diff
@@ -350,20 +356,46 @@ public class ${className} implements Parcelable {
 #if($field)
 #if(${field.getIsArrayType()})
         if (${field.getPrivateVariableName()} == null) {
+#if($useBcLogger)
```

Comment:
There's a bit of duplicated code here in all these ```if($useBcLogger) / else``` blocks that I'm not happy about, but I guess it's OK.